src/utils/time.ts

```diff
@@ -167,9 +197,7 @@ export const daysFromStartOfEvent = (utcSeconds: number) => {
 };
 
 export const dateEventTimeFormat = (date: Date) => {
-  const hh = String(date.getHours()).padStart(2, "0");
-  const mm = String(date.getMinutes()).padStart(2, "0");
-  return hh + ":" + mm;
+  return format(date, "HH:mm");
 };
```

Comment:
This seems very similar to `formatHourAndMinute` (except that it accepts a `Date` rather than a `number`)

Might be worth adding an `@debt` comment calling this out, so that we can refactor them in future.

Reply:
instead of adding `@debt` comment, I've searched for the usage of `dateEventTimeFormat` across the project, and I've found only 1 place where seconds were transformed to a Date object, so I just refactored that as well. and removed `dateEventTimeFormat`.

I haven't added `Date` as a possible type to `formatHourAndMinute` which accepts only utcSeconds as a parameter right now. I think we can do it when needed. 

and there was a function `getCurrentTimeInUnixEpochSeconds` which was the same as `getCurrentTimeInUTCSeconds`. I've removed it as well, and replaced all its mentionings by `getCurrentTimeInUTCSeconds`.

Reply:
Sounds good :)